### ISSUE TITLE
"secretsFile" in documents is "secretFile" in code

### ISSUE THREAD

**Issue by hatemosphere** [Praqma/helmsman#105]
Hey guys. I was desperately trying to use helmsman's new integration with helm-secrets plugin. Apparently, there is a mismatch between documentation here https://github.com/Praqma/helmsman/blob/master/docs/desired_state_specification.md and actual code here https://github.com/Praqma/helmsman/blob/master/release.go#L22-L23
I feel like the code needs to be aligned with doc, not in the opposite way since we have Helm "values" = "valuesFile(s)", so Helm "secrets" IMO should be equal "secretsFile(s)".

**Comment by sami-alajrami** [comment]
Hi @hatemosphere it is secretsFile(s) in both code and docs :) 

Here is the docs part (under [Apps](https://github.com/Praqma/helmsman/blob/master/docs/desired_state_specification.md#apps) section)
```
secretsFile : a valid path to custom Helm secrets.yaml file. File extension must be yaml. Cannot be used with secretsFiles together. Leaving it empty uses the default chart secrets.
secretsFiles : array of valid paths to custom Helm secrets.yaml file. File extension must be yaml. Cannot be used with secretsFile together. Leaving it empty uses the default chart secrets.
```

Or, am I missing something?

**Comment by hatemosphere** [comment]
@sami-alajrami could you please check the link to the codebase I posted above? it's "secretFile" in a code.

**Comment by sami-alajrami** [comment]
Oh .. now I see the missing (s). I agree, it should be `secretsFile`. Thanks for spotting this!

**Comment by sami-alajrami** [comment]
this is shipped in v1.7.0
